examples/axi_lite_slave/tests/Makefile

```diff
@@ -6,7 +6,6 @@ COCOTB=$(PWD)/../../..
 PYTHONPATH := ./model:$(PYTHONPATH)
 
 export PYTHONPATH
-export PYTHONHOME=$(shell python -c "from distutils.sysconfig import get_config_var; print(get_config_var('prefix'))")
```

Comment:
i don't know why I missed this, maybe because my docker images were either python 2.7 or 3.6, but not both?

Reply:
Could certainly be. I guess we'll run into more problems like that until cocotb gets real Python packaging (bug #517) [but that will require much more work to get right, so fixing the fallout for now]